user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection/Providing_feedback_on_behavioral_anomalies.md

```diff
@@ -68,7 +93,7 @@ After you have provided feedback on a suggestion event or alarm, a light bulb ic
 | Action | Description |
 |--|--|
 | Clear event | Clears the suggestion event immediately. No confirmation box will appear. |
-| Improve alarm template | Opens a pop-up window suggesting changes to the alarm template, based on previous feedback for the parameter. At the top of the window, you can see the element and parameter name, along with a link to relevant documentation<!--RN 39616-->. At the bottom, it shows which elements using the same alarm template will be affected by the suggested template changes<!--RN 39729-->. A clock button allows you to view the current anomaly template configuration<!--RN 39640-->. |
+| Improve alarm template | Opens a pop-up window suggesting changes to the alarm template, based on previous feedback for the parameter. At the top of the window, you can see the element and parameter name, along with a link to relevant documentation<!--RN 39616-->. At the bottom, it shows which elements using the same alarm template will be affected by the suggested template changes<!--RN 39729-->. A clock button allows you to view the current template configuration<!--RN 39640-->. |
 | Create alarm template | Opens a card to create a new alarm template from scratch. |
```

Comment:
This now also opens the same pop-up as above, but with a button 'Create alarm template' instead of 'Update alarm template', and without the clock icon to show the previous configuration (as there is no previous configuration). Clicking 'Create alarm template' will then create a new alarm template with the suggested configuration and apply it to the current element

Reply:
```suggestion
| Create alarm template | Opens a card to create a new alarm template from scratch. From DataMiner 10.5.1/10.6.0 onwards<!--RN 41451-->, this follow-up action is also available after providing feedback on suggestion events or alarms generated by proactive cap detection. In such cases, a suggested configuration is provided based on previous feedback for the parameter.  |
```

Is this accurate, @tobedeprez?

Reply:
Not completely. In older Cube versions (prior to 10.5.1/10.6.0 **and** without automatic Cube updates), we would indeed open a card to create a new alarm template, but without any proposed configuration. Since 10.5.1/10.6.0, or with automatic Cube updates, the behavior has changed, also for feedback on anomalies. Since then, we do no longer use a card, but instead we show the same pop-up as with 'Improve alarm template', but with the differences mentioned above (i.e. other button and no clock button for previous configuration).

Reply:
```suggestion
| Create alarm template | Opens a pop-up window that suggests an alarm template configuration based on previous feedback. At the top of the window, the element and parameter name are displayed, along with a link to the relevant documentation. Prior to DataMiner 10.5.1/10.6.0<!--RN 41451-->, clicking this follow-up action will open a card to create a new alarm template from scratch. |
```

Thanks for the clarification. I think this may be better then, @tobedeprez.